cluster-autoscaler/utils/nodegroupset/compare_nodegroups.go

```diff
@@ -55,6 +55,13 @@ func compareResourceMapsWithTolerance(resources map[apiv1.ResourceName][]resourc
 // are similar enough to likely be the same type of machine and if the set of labels
 // is the same (except for a pre-defined set of labels like hostname or zone).
 func IsNodeInfoSimilar(n1, n2 *schedulercache.NodeInfo) bool {
+	// This is GKE specific logic. If both nodes come from same GKE node pool
+	// treat them as similar
+	// TODO move this logic to cloud provider interface some how
```

Comment:
Is this TODO really necessary? We've already fixed it in all later versions by moving it to a processor.